sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/models/CreateQueueOptions.java

```diff
@@ -462,4 +464,28 @@ public CreateQueueOptions setUserMetadata(String userMetadata) {
         this.userMetadata = userMetadata;
         return this;
     }
+
+    /**
+     * Get the maxMessageSizeInKilobytes property: The maximum size of a message in kilobytes.
+     *
+     * @return the maxMessageSizeInKilobytes value.
+     */
+    public long getMaxMessageSizeInKilobytes() {
+        return this.maxMessageSizeInKilobytes;
+    }
+
+    /**
+     * Set the maxMessageSizeInKilobytes property: Represents the default maximum message size (in kilobytes)
+     * Option only available in premium tier. Default maximum in Standard tier is 256 KB, and 1 MB in premium tier.
+     * Larger message sizes are available in preview.
+     * Please see <a href=https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-premium-messaging#large-messages-support-preview>more info</a>
```

Comment:
nit: remove en-us so user's browser can discern which language to use.

Reply:
nice! didn't know this neat trick

Reply:
it should actually fail CI if we have any embedded URIs to language-specific docs sites